mrp_partial_production/__manifest__.py

```diff
@@ -0,0 +1,27 @@
+# coding: utf-8
+# Copyright 2017 Vauxoo (https://www.vauxoo.com) <info@vauxoo.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+{
+    "name": "MRP Partial Production",
+    "version": "10.0.0.1.0",
+    "author": "Vauxoo",
+    "category": "",
+    "website": "http://www.vauxoo.com/",
+    "license": "LGPL-3",
+    "depends": [
```

Comment:
could you check the depends?

I think are not necessary all

Reply:
I removed the overlapping depend 'stock'